docs/Displaying a User Interface/Adaptive Interface Capabilities/index.md

```diff
@@ -94,8 +103,14 @@ boolean navigationSupported = sdlManager.getSystemCapabilityManager().isCapabili
 ```
 !@
 
+@![javascript]
+```js
+const navigationSupported = sdlManager.getSystemCapabilityManager().isCapabilitySupported(SDL.rpc.enums.SystemCapabilityType.NAVIGATION);
```

Comment:
That's the required hierarchy to get the enum. You could define the enum earlier in the file to avoid typing it out every time, but you would still need this somewhere.